src/ITILSolution.php

```diff
@@ -220,6 +220,9 @@ public function prepareInputForAdd($input)
             $input = array_replace($template_fields, $input);
         }
 
+        if (isset($input['_do_not_compute_status'])) {
+            return $input;
```

Comment:
There is a piece of code that is not related to status on line 264. Instead of returning here it would be preferable to encapsulate the whole status update logic in a `if (!this->item->isStatusComputationBlocked($input)) {}` block.